man/docker-cp.1.md

```diff
@@ -78,7 +78,8 @@ you must be explicit with a relative or absolute path, for example:
     `/path/to/file:name.txt` or `./file:name.txt`
 
 It is not possible to copy certain system files such as resources under
-`/proc`, `/sys`, `/dev`, and mounts created by the user in the container.
+`/proc`, `/sys`, `/dev`, tmpfs, and mounts created by the user in the container.
```

Comment:
Done. 

I added the example to the `#DESCRIPTION` section rather than the `#EXAMPLES` section, because it's an example for `docker exec` rather than `docker cp` it self.

If this seems ugly, I suggest reverting to my previous commit https://github.com/docker/docker/pull/22900/commits/b78b3a17fa6306cf2d2941aaeaf98a3037304b06

Thanks!